src/netaddress.cpp

```diff
@@ -1187,7 +1187,11 @@ std::string CSubNet::ToString() const
             if (netmask[i] == 0x00) {
                 break;
             }
-            cidr += NetmaskBits(netmask[i]);
+            const int num_bits{NetmaskBits(netmask[i])};
+            if (num_bits == -1) {
```

Comment:
Will update per https://github.com/bitcoin/bitcoin/pull/22586#issuecomment-889504755 (thanks @tryphe!) if CI is green.
```suggestion
            if (num_bits == -1) {
                // Invalid subnet mask; -1 value is incompatible with uint8_t cidr.

```